### PR TITLE
feat(app): App robot settings other tabs robot is busy

### PR DESCRIPTION
**Pull request by koji**
# Overview
This will fix the rest of #10322  (Networking tab & Calibration tab)

Details
https://github.com/Opentrons/opentrons/pull/10368

# Changelog
- Added functionality that checks whether a robot is busy or not to RobotSettings - Networking tab
- Added functionality that checks whether a robot is busy or not to RobotSettings - Calibration tab
- Added test cases to each tab

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] solve conflicts

File: app/src/organisms/Devices/RobotSettings/TemporarySelectNetwork.tsx [hunk 1]
```diff
@@ -87,26 +86,37 @@ export const TemporarySelectNetwork = ({
...
+      updateRobotStatus(true)
+    } else {
+      const ssid = activeNetwork?.ssid
+      ssid != null && setChangeState({ type: DISCONNECT, ssid })
+    }
   }
 
   const handleSelectJoinOther = (): void => {
-    setChangeState({ type: JOIN_OTHER, ssid: null })
+    console.log('handleSelectJoinOther need to check robotStatus')
```
Comment (b-cooper) [inline app/src/organisms/Devices/RobotSettings/TemporarySelectNetwork.tsx]:
```suggestion
```

**Review by b-cooper — APPROVED** [review]
💹

**Comment by koji** [comment]
- [x] address pr comment
- [x] merge #10368 into edge
- [x] change the merge direction to `edge` and solve conflicts
- [ ] merge

**Pull request merged**